.github/workflows/release_to_stable.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release all FINOS Legend charms to stable
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: "The release to promote from edge to stable (yyyy-mm-dd)"
+        type: string
+        required: true
+jobs:
+  release-to-stable:
+    name: Release FINOS Legend charms from edge to stable
+    runs-on: ubuntu-latest
+    env:
+      CHARMCRAFT_AUTH: "${{ secrets.CHARMHUB_TOKEN }}"
+    steps:
+      - name: Install Dependencies
+        run: |
+          sudo snap install jq
+          sudo snap install charmcraft --classic
+
+      - name: Release to stable
+        run: |
+          release_short=$(echo ${{ github.event.inputs.release }} | cut -d- -f1-2)
+
+          release_charm_to_stable(){
+            charm_name="$1"
+            charm_resource="$2"
+
+            charm_status=$(charmcraft status --format json "${charm_name}")
+            release_info=$(echo "$charm_status" | jq -r '.[].mappings[].releases[] | select(.channel == "'"$release_short"'/edge")')
+            charm_rev=$(echo "$release_info" | jq -r '.revision')
+            release_channel="$release_short"
+
+            # if we don't have release info for the specified channel, we'll use the latest edge revision
+            if [ -z "$release_info" ]; then
+              release_info=$(echo "$charm_status" | jq -r '.[].mappings[].releases[] | select(.channel == "latest/edge")')
+              charm_rev=$(echo "$release_info" | jq -r '.revision')
+              release_channel="latest"
+            fi
+
+            # if we have a charm resource we get that resource's revision
+            if [ -n "$charm_resource" ]; then
+              resource_rev=$(echo "$release_info" | jq -r '.resources[] | select(.name == "'"$charm_resource"'") | .revision')
+              # release the charm to the the corresponding stable channel with the same charm revision and the same charm resource revision
+              charmcraft release "${charm_name}" --revision=${charm_rev} --channel="${release_channel}/stable" --resource="${charm_resource}:${resource_rev}" --verbose
+            else
+              # release the bundle to the corresponding stable channel with the same charm revision
+              charmcraft release "${charm_name}" --revision=${charm_rev} --channel="${release_channel}/stable" --verbose
+            fi
+          }
+
+          charmcraft whoami
+
+          release_charm_to_stable finos-legend-bundle
```

Comment:
So, in the Legend bundle we also specify the channel of the charms we're deploying. For example, for the ``edge`` bundle, we deploy the ``edge`` charms. Simply releasing that bundle into the ``stable`` track won't update those values automatically, the bundle will still deploy ``edge`` charms (https://github.com/finos/legend-juju-bundle/blob/abdcd77a54225d01dbd303aba382a46bd43dc029/bundle.yaml#L19). For the bundle, we should instead create an overlay to override the charms' channels, similarly to how the release to edge works.

Reply:
@claudiubelu I have added steps for updating the `bundle.yaml` file and uploading the bundle to stable.